core/store/src/trie/mod.rs

```diff
@@ -103,7 +102,7 @@ pub enum KeyLookupMode {
 const TRIE_COSTS: TrieCosts = TrieCosts { byte_of_key: 2, byte_of_value: 1, node_cost: 50 };
 
 #[derive(Clone, Copy, Hash)]
-pub(crate) enum ValueHandle {
+pub enum ValueHandle {
```

Comment:
Needed to make this `pub` from `pub(crate)` as `disk_iter` functions is exposed outside module